src/plugins/guided_onboarding/public/components/guide_button_popover.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { useEffect, useRef } from 'react';
+import { EuiPopover, EuiText, EuiPopoverTitle } from '@elastic/eui';
```

Comment:
Ideally, we would use `EuiTour` here instead of `Popover` but I realized something when checking this, in Figma the `EuiTour` component lets you hide the footer via a prop. But looking at the EUI documentation it doesn't seem like that prop exists in the coded component. Was this the reason to use `Popover` as it matches stylistically to the design doc? 

If so, are we able to hide the footer on these types of tour steps? (Not sure if we are able to differentiate them, let me know what you think, and if it adds too much complexity for this styling).